### DISCUSSION TITLE
About implementing headless SwapChain

### DISCUSSION THREAD

**Discussion by kmw4988** [google/filament#5474] [Q&A] [answered]
Hi, filament team.

I leave it here because there is no suitable option for the issues page.

Headless swapchain `createSwapChain(uint32_t width, uint32_t height, uint64_t& flags)` is not implemented on PlatformCocoaTouchGL.mm.

I'm already implementing and using it.

Can I send a pull request?

**Comment by bejado** [comment, marked as answer]
You're welcome to send a PR. We're trying to deprecate GL on iOS so depending on the complexity we may or may not accept it.
